getting_started/first-app.rst

```diff
@@ -20,3 +20,15 @@ If everything goes well you should see some front-ends and back-ends showing up
 - http://stilton.yourdomain.com
 - http://cheddar.yourdomain.com
 - http://wensleydale.yourdomain.com
+
+One simply and quick way to access the Traefik UI is to tunneling via ssh to one of the edge nodes with the following command::
+
+    ssh -N -f -L localhost:808x:localhost:808x ubuntu@<edge-floating-ip>
+
+If you are wondering how to get the `edge-floating-ip`, then open the `inventory` file which is created after having spun up the cluster in the KubeNow folder
+
+By visitng your localhost:808x in your browser you should be displayed with the following UI:
+
+.. image:: ../img/traefik_UI_example.png
```

Comment:
Absolutely. Reason why I decided to use this one was that front-end and back-end section are clearly stated compared to KubeNow. However good catch regarding whether or not we could reuse this image. Working on it.